### PR TITLE
[fa] Update inline validation file

### PR DESCRIPTION
**Pull request by Kamandlou**
update inline validation file

### CODE REVIEW

File: locales/fa/validation-inline.php [hunk 1]
```diff
@@ -36,8 +36,8 @@
     'date'                 => 'این مقدار یک تاریخ معبتر نیست.',
     'date_equals'          => 'این مقدار باید یک تاریخ مساوی با :date باشد.',
     'date_format'          => 'این مقدار با فرمت :format همخوانی ندارد.',
-    'declined'             => 'This value must be declined.',
-    'declined_if'          => 'This value must be declined when :other is :value.',
+    'declined'             => 'این مقدار قابل پذیرش نیست',
```
Comment (andrey-helldar) [inline locales/fa/validation-inline.php:39]: We missed the period at the end of the sentence.

Comment (Kamandlou) [reply locales/fa/validation-inline.php:39]: I will fix it

File: locales/fa/validation-inline.php [hunk 2]
```diff
@@ -13,15 +13,15 @@
...
+    'accepted_if'          => 'این مقدار باید پذیرفته شود زمانه که :other برابر با :value.',
     'active_url'           => 'این مقدار یک آدرس معتبر نیست.',
     'after'                => 'این مقدار باید یک تاریخ بعد از :date باشد.',
     'after_or_equal'       => 'این مقدار باید یک تاریخ مساوی یا بعد از :date باشد.',
     'alpha'                => 'این مقدار تنها میتواند شامل حروف باشد.',
     'alpha_dash'           => 'این مقدار تنها میتواند شامل حروف، اعداد، خط تیره و زیرخط باشد.',
     'alpha_num'            => 'این مقداز تنها میتواند شامل حروف و اعداد باشد.',
     'array'                => 'این مقدار باید یک آرایه باشد.',
-    'attached'             => 'This field is already attached.',
+    'attached'             => 'این فیلد قبلاً پیوست شده است.',
```
Comment (andrey-helldar) [inline locales/fa/validation-inline.php:24]: The attached key belongs to Laravel Nova and we moved it to another file: https://github.com/Laravel-Lang/lang/blob/master/locales/fa/validation-nova-inline.php#L15

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Comment by andrey-helldar** [comment]
@Kamandlou, can you translate a [json](https://laravel-lang.github.io/lang/statuses/fa.html) keys too?

**Comment by Kamandlou** [comment]
> @Kamandlou, can you translate a [json](https://laravel-lang.github.io/lang/statuses/fa.html) keys too?

Yes, I can.

File: locales/fa/validation-inline.php (hunk 2 above)
Comment (Kamandlou) [reply locales/fa/validation-inline.php]: ok :)

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
Thank you!

**Comment by Kamandlou** [comment]
> Thank you!

you're welcome :)

**Pull request merged**